### PR TITLE
added contact tab to navbar

### PR DESCRIPTION
**Pull request by KeatonSushi**
nothing fancy. just takes the user straight to gmail. not sure how to use php yet.

### CODE REVIEW

**Review by LitterPic — COMMENTED** [review]
what are the .webp files you're adding?

File: aboutUsNew.html [hunk 1]
```diff
@@ -1,86 +1,251 @@
...
+    <!-- google font stuff -->
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+    />
+
+    <script src="index.js" defer></script>
+  </head>
+  <body>
+    <nav class="navbar navbar-expand-lg">
```
Comment (LitterPic) [inline aboutUsNew.html:35]: same question as the other MR, are you replacing the navbar I created?

File: styles.css [hunk 2]
```diff
@@ -239,3 +246,87 @@ header .icons span:last-child {
...
+
+#logos-div {
+  display: flex;
+  justify-content: center;
+  margin: 100px;
+}
+
+.donate-btn {
+  background-color: black;
+  animation: color-change 1s infinite;
```
Comment (LitterPic) [inline styles.css:319]: Please remove any blinking text/animation.  Nothing should blink on the webpage; it distracts end users and does not have the desired effect.  Thanks!

**Comment by KeatonSushi** [comment]
The seals from the original donate page.

**Review by LitterPic — APPROVED** [review]

**Pull request merged**